airflow/decorators/__init__.pyi

```diff
@@ -251,5 +251,120 @@ class TaskDecoratorCollection:
         :param cap_add: Include container capabilities
         """
         # [END decorator_signature]
+    def kubernetes(
+        self,
```

Comment:
```suggestion
    def kubernetes(
        self,
        *,
```

Arguments on `KubernetesOperator` are keyword-only, so these need to be as well.